### PR TITLE
Error reason

### PR DESCRIPTION
**Pull request by eboisgon**
Would allow to acces more easily the erro reason 

For example : 

```
begin 
	variable = Mailjet::Campaigndraft_send.create(id: 3585077)
	pp variable
rescue Mailjet::ApiError => e
    p "error : #{e.code} , reason : #{e.reason}"
end
```

would give this 

"error : 400 , reason : Newsletter has to be in status draft or programmed"

if the campaignDraft has already been sent

### CODE REVIEW

File: lib/mailjet/api_error.rb [hunk 1]
```diff
@@ -16,7 +19,7 @@ def initialize(code, res, request, request_path, params)
           end.join("\n")
         else
           res.inspect
-        end + "\n\nPlease see http://api.mailjet.com/0.1/HelpStatus for more informations on error numbers.\n\n"
+        end + "\n\nPlease see https://dev.mailjet.com/guides/#status-codes for more informations on error numbers.\n\n"
```
Comment (arnaudbreton) [inline lib/mailjet/api_error.rb]: @eboisgon good catch!

**Comment by neolyte** [comment]
Merci Arnaud :)

**Comment by arnaudbreton** [comment]
Hey @neolyte, you're welcome! Apologies for the delay to merge this PR... Seems we lagged behind here. Anyway, version 1.4.11 is now live on rubygems.com. 

Let us know if everything's OK for you.
Cheers

**Comment by neolyte** [comment]
Seems to be breaking the save action on campaigns now:

Example:
`
x = Mailjet::Newsletter.find(campaign_id)
x.save
`

Yield:
`MultiJson::ParseError: JSON::ParserError`

**Comment by neolyte** [comment]
Everything works well when updating api_error.rb with the following:

```
unless res.blank?
        resdec = ActiveSupport::JSON.decode(res)
        self.reason = resdec['ErrorMessage']
end
```

**Pull request merged**